modules/geo-layers/src/terrain-layer/terrain-layer.js

```diff
@@ -65,6 +65,18 @@ function urlTemplateToUpdateTrigger(template) {
   return template;
 }
 
+// Get [min, max] elevation range of terrain mesh
+function getZRange(positions) {
```

Comment:
I'll add the bounding box generation to the loader so that we only do this once for each tile.